src/EA.Weee.DataAccess/StoredProcedure/EvidenceStoredProcedures.cs

```diff
@@ -23,13 +24,19 @@ public async Task<List<AatfEvidenceSummaryTotalsData>> GetAatfEvidenceSummaryTot
             return await context.Database.SqlQuery<AatfEvidenceSummaryTotalsData>(queryString, aatfIdParameter, complianceYearParameter).ToListAsync();
         }
 
-        public async Task<List<ObligationEvidenceSummaryTotalsData>> GetObligationEvidenceSummaryTotals(Guid pcsId, int complianceYear)
+        public async Task<List<ObligationEvidenceSummaryTotalsData>> GetObligationEvidenceSummaryTotals(Guid? pcsId, Guid? orgId, int complianceYear)
         {
-            string queryString = "[Evidence].[getObligationEvidenceSummaryTotals] @SchemeId, @ComplianceYear";
-            SqlParameter pcsIdParameter = new SqlParameter("@SchemeId", pcsId);
+            string queryString = "[Evidence].[getObligationEvidenceSummaryTotals] @ComplianceYear, @OrganisationId, @SchemeId ";
```

Comment:
@GuidoWorkDev Can you add a Condition check to ensure that the orgId and the pcs id are not null at the same time